### PR TITLE
presence validations added to models

### PR DESCRIPTION
**Pull request by killeraliens**
Updated the 3 models with presence validations

### CODE REVIEW

File: app/models/booking.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 class Booking < ApplicationRecord
   belongs_to :user
   belongs_to :task
+  validates  :user, :task, presence: true
```
Comment (mgould16) [inline app/models/booking.rb]: do user and task go on separate lines? not sure, looking for validation from the group here for my learning.

Comment (killeraliens) [reply app/models/booking.rb]: nope you can put all of them on single line:)

Comment (jamero102) [inline app/models/booking.rb]:
```suggestion

```

File: app/models/task.rb [hunk 2]
```diff
@@ -1,4 +1,5 @@
 class Task < ApplicationRecord
   belongs_to :user
   has_many :bookings
+  validates :user, :title, :description, :start_date, :end_date, :start_time, :end_time, :category, presence: true
```
Comment (jamero102) [inline app/models/task.rb]:
```suggestion
  validates :title, :description, :start_date, :end_date, :start_time, :end_time, :category, presence: true
```

File: app/models/booking.rb (hunk 1 above)
Comment (killeraliens) [reply app/models/booking.rb]: ok this makes sense

File: app/models/task.rb (hunk 2 above)
Comment (killeraliens) [reply app/models/task.rb]: ok this makes sense

**Comment by jamero102** [comment]
looks good now!

**Comment by MaddieLewis** [comment]
I think this looks great

**Pull request merged**